core/common/src/main/java/alluxio/extensions/ExtensionsClassLoader.java

```diff
@@ -82,4 +84,14 @@ public ExtensionsClassLoader(URL[] urls, ClassLoader defaultClassLoader) {
       return mDefaultClassloader.loadClass(name, resolve);
     }
   }
+
+  @Override
+  public Enumeration<URL> getResources(String name) throws IOException {
+    Enumeration<URL> resources = super.getResources(name);
+    // Falls back to default class loader if not found in the URL class loader
+    if (!resources.hasMoreElements()) {
+      return mDefaultClassloader.getResources(name);
```

Comment:
Is there a rare enough/significant enough event that we want to log something?

Reply:
Since it can be used by 3rd party component we can't really assume the pattern/frequency. I can add a debug log in case we want to track it.

Reply:
Got it, let's leave something at DEBUG level at least so we have insight if something is unexpected. If you think the frequency is likely minimal, we can bump it up to INFO.